### PR TITLE
Patches: Add widescreen, 60fps and authors

### PR DESCRIPTION
**Pull request by fox140cv**
All patches have been tested and work correctly.
Add some authors, visually it is better and more organized than "Unknown"

### CODE REVIEW

File: patches/SLES-53030_72DC82B5.pnach [hunk 1]
```diff
@@ -1,9 +1,9 @@
-gametitle=Hitman - Blood Money (PAL-G) (SLES-53030)
+72DC82B5gametitle=Hitman - Blood Money (PAL-G) (SLES-53030)
```
Comment (pgert) [inline patches/SLES-53030_72DC82B5.pnach:1]: typo error

File: patches/SLES-53191_F7780E06.pnach [hunk 2]
```diff
@@ -3,7 +3,7 @@ gametitle=Kaido Racer PAL (SLES_531.91)
 [Widescreen 16:9]
 gsaspectratio=16:9
 author=El_Patas
-comment=Widescreen hack by El_Patas
+comment=Widescreen Hack
```
Comment (pgert) [inline patches/SLES-53191_F7780E06.pnach:6]: not by El_Patas?

File: patches/SLES-54490_DE9722A5.pnach [hunk 3]
```diff
@@ -2,8 +2,8 @@ gametitle=God Hand (PAL-M5) (SLES-54490)
 
 [Widescreen 16:9]
 gsaspectratio=16:9
-comment=Widescreen Hack (16:9) by ElHecht and Arapapa
-
+author=Arapapa
+comment=Widescreen Hack
```
Comment (pgert) [inline patches/SLES-54490_DE9722A5.pnach:6]: not by ElHecht and Arapapa?

File: patches/SLES-53191_F7780E06.pnach (hunk 2 above)
Comment (fox140cv) [reply patches/SLES-53191_F7780E06.pnach:6]: It is created by El_Patas, I have tried to be thorough. I didn't want to be redundant by also including the author in the description.
https://forums.pcsx2.net/Thread-PCSX2-Widescreen-Game-Patches?pid=360707#pid360707

File: patches/SLES-53030_72DC82B5.pnach (hunk 1 above)
Comment (fox140cv) [reply patches/SLES-53030_72DC82B5.pnach]: fixed, thanks!

File: patches/SLES-54490_DE9722A5.pnach (hunk 3 above)
Comment (fox140cv) [reply patches/SLES-54490_DE9722A5.pnach:6]: fixed, thanks pgert!, I have tried to be very thorough, I will go over them all again

**Comment by fox140cv** [comment]
Reviewed them all again, those 2 were the only errors. Thanks again pgert!

**Comment by refractionpcsx2** [comment]
Thanks, that's a lot of patch changes! Appreciate your efforts

**Pull request merged**